package.json

```diff
@@ -92,6 +92,7 @@
     "minimist": "^1.2.0",
     "mongoose": "^4.9.8",
     "morgan": "^1.8.1",
+    "ms": "^2.0.0",
```

Comment:
This is the same package used by the JWT token generator. It ensures that the same logic is applied when parsing the duration. There are some subtle parsing differences with `parse-duration` and the `ms`.